### DISCUSSION TITLE
`brew tap <org>/<tap> git@github.com:<org>/homebrew-<tap>` broken?

### DISCUSSION THREAD

**Discussion by nickatsegment** [Homebrew/discussions#887] [Everyday usage] [answered]
When you tap a private tap, brew says you should set its origin with `cd <dir>; git remote set-origin ...blah blah blah SSH...` to use the SSH protocol so you don't have to put your password in. That's what I want to do with `brew tap <url>`, but it tells me the repo names don't match :(

**Comment by Rylan12** [comment, marked as answer]
I can't seem to reproduce this. It's likely an issue with the specific command you're running. Since its a private repo and I'd assume you wouldn't want to share the name/url (of course), it's hard to give specific feedback about what's not working. Here's an example that _does_ work using [one of my test public repos](https://github.com/Rylan12/homebrew-personal). I've tested the same command on a private repo, too.

```console
$ brew tap rylan12/personal git@github.com:Rylan12/homebrew-personal.git
==> Tapping rylan12/personal
Cloning into '/usr/local/Homebrew/Library/Taps/rylan12/homebrew-personal'...
remote: Enumerating objects: 84, done.
remote: Counting objects: 100% (84/84), done.
remote: Compressing objects: 100% (58/58), done.
remote: Total 84 (delta 15), reused 79 (delta 13), pack-reused 0
Receiving objects: 100% (84/84), 13.37 KiB | 1.91 MiB/s, done.
Resolving deltas: 100% (15/15), done.
Tapped 2 formulae (32 files, 51.7KB).
```

**Reply by nickatsegment** [reply]
Thanks for testing :) Let me see if I can repro with something small and non-sensitive

**Comment by nickatsegment** [comment]
Well damn, I can't repro with my test repo, and indeed it now works with my original tap. Whoops
